.github/workflows/test-python.yml

```diff
@@ -65,22 +69,26 @@ jobs:
           net start MongoDB
       - name: Install libbson
         run: |
-          LIBBSON_INSTALL_DIR=$(pwd)/libbson ./build-libbson.sh
+          ./build-libbson.sh
       - name: Install Python dependencies
         run: |
           python -m pip install -U pip
       - name: Install pymongoarrow
         run: |
           # Install the library with no deps
-          LIBBSON_INSTALL_DIR=$(pwd)/libbson python -m pip install -vv -e "."
+          python -m pip install -vv -e "."
       - name: Ensure imports with no test deps
         run: |
           python -c "from pymongoarrow.monkey import patch_all; patch_all()"
       - name: Run tests
         run: |
           # Install the test deps
-          LIBBSON_INSTALL_DIR=$(pwd)/libbson python -m pip install -vv -e ".[test]"
-          LD_LIBRARY_PATH=$(pwd)/libbson/lib python -m unittest discover test -v
+          python -m pip install -vv -e ".[test]"
+          python -m unittest discover test -v
+      - name: Run the tests with pytest
+        run: |
+          pip install pytest
+          pytest -vv
```

Comment:
I don't think it makes sense to run the test suite twice. We should only support and only run one test framework.

Reply:
It only takes 21 seconds, and I didn't want to impose `pytest`, but it is largely compatible with `unittest`.  This guard was meant to ensure that either would continue to work.

Reply:
It still is not worthwhile to maintain compatibility with two different test runners. If we want to switch to pytest then we should drop unittest and only use pytest (not that I'm suggesting it).